release.nix

```diff
@@ -0,0 +1,4 @@
+let
```

Comment:
See https://mmhaskell.com/blog/2020/2/10/converting-cabal-to-nix for more context